README.md

```diff
@@ -1,13 +1,36 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+In this project I have created a form in JS React by mounting the form elements in components and using state! 
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
+In this project I started of with creating the input elements in the form in different components. Working with mounting the code in components makes it more logical for me. I created text inputs, radio buttons, a dropdown and finally a range slider - all connected to a form element. The input from the form was connected into a summary that is visible for the user when the last input is submitted. 
+
+Early in the project I added the different components into different sections, showing the questions one by one, and I added buttons making the user able to go forward and back in the form. 
+
+The hardest part for me has been that the required-attributes doesn't work when the form elements are mounted in different components. It does work in Firefox but not in Chrome, Safari or Edge. To make the user answer before being able to click on the 'Next'-button I disabled the buttons until the question was answered - except for the radio buttons. 
```

Comment:
Hmm.. I had no problems with this. On the select and input - questions just adding required worked, and on the radiobuttons the important part was just to add a "name" and "id" - attribute and then required in the input-tag. 
But on the other hand I didn't do the requirement with the questions in different sections and next-buttons. Could that have something to do with it? Can we ask one of the code coaches to clarify? :) 

Good that you found your own solution anyway! 👍